src/main/java/mate/academy/spring/controller/AuthenticationController.java

```diff
@@ -0,0 +1,31 @@
+package mate.academy.spring.controller;
+
+import mate.academy.spring.security.AuthenticationService;
+import mate.academy.spring.service.UserService;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/register")
+public class AuthenticationController {
+    private final UserService userService;
+    private final AuthenticationService authenticationService;
+
+    public AuthenticationController(UserService userService,
+                                    AuthenticationService authenticationService) {
+        this.userService = userService;
+        this.authenticationService = authenticationService;
+    }
+
+    @PostMapping
+    public boolean register(@RequestParam String email, @RequestParam String password) {
+        if (userService.findByEmail(email).isPresent()) {
+            return false;
+        } else {
```

Comment:
do we need else after return?

and I would recommend throwing an exception rather than returning false